package.json

```diff
@@ -23,7 +23,11 @@
     "@angular/ssr": "^17.3.7",
     "@ng-icons/bootstrap-icons": "^27.4.0",
     "@ng-icons/core": "^27.4.0",
+    "emoji-toolkit": "^8.0.0",
     "express": "^4.19.2",
+    "marked": "^12.0.2",
+    "ngx-markdown": "^18.0.0",
```

Comment:
use version 17.2.1

version 18.x.x is for Angular 18 and we're still on 17

Reply:
Changed were made